models/brac/views/formview_postnatal_care.sqlx

```diff
@@ -1,43 +0,0 @@
-SELECT COALESCE((regexp_split_to_array((form.doc #>> '{fields,geolocation}'::text[]), ' '::text))[1], (form.doc #>> '{fields,location,lat}'::text[])) AS "inputs/meta/location/lat",
```

Comment:
Are these files no longer needed?

Reply:
These files are not dependencies of any files. There are `useviews_` for many of these forms which select directly from couchdb so I removed these unneeded and redudant files.